AndelaOSP/Andela-Socials
# #166491401 Fix calendar invite flow

#### What Does This PR Do?
This PR fixes the calendar invite flow when user attends an event from the application

#### Description Of Task To Be Completed
- Fix duplication of event when the user attends an event from the application
- Mark the event as accepted when the user accepts the event through the application
- Add tests

#### Any Background Context You Want To Provide?
None

#### How can this be manually tested?
- Fetch the branch locally by running `git fetch origin bg-fix-calendar-invite-flow-166491401` & `git checkout bg-fix-calendar-invite-flow-166491401` respectively
- Create an event in the application
- Let another user login to the application and attend the event created earlier in the application. The event should be added into the attendee's calendar and also updated to attending

#### What are the relevant pivotal tracker stories?
[#166491401](https://www.pivotaltracker.com/story/show/166491401)
